introduction.md

```diff
@@ -6,41 +6,170 @@ layout: documentation
+
+During the definition of your [Thing]({{base}}/concepts/things) you will identify the channel to which your [Item]({{base}}/concepts/items) will be linked. 
+These two steps ensure that openHAB can transmit the information from the [Thing]({{base}}/concepts/things) to the [Item]({{base}}/concepts/items) (and vice versa).
+
+### Bindings
+
+Bindings are software packages that are installed by the user in openHAB. 
+The main purpose of Bindings are to establish the connection between your device and your [Thing]({{base}}/concepts/things). 
+Bindings communicate with your device and translate all commands to and from openHAB between your device and your [Thing]({{base}}/concepts/things).
+
+Bindings are provided at the [Add-on section](https://www.openhab.org/addons/) of the [openHAB website](https://www.openhab.org). 
+Here you will find a searchable list of several hundred bindings to support as many devices as possible. 
+New bindings are regularly added as developers integrate more devices into openHAB.
+
+For each binding, detailed instructions and examples are provided that include guidance on configuration (if any) of the binding itself, the definition of [Things]({{base}}/concepts/things) supported by this binding and the Channels these [Things]({{base}}/concepts/things) provide. 
+In most cases, the description also contains a fully worked out example that includes a definition of [Things]({{base}}/concepts/things) and its Channels, [Items]({{base}}/concepts/items) linked to those Channels and the use of these [Items]({{base}}/concepts/items) in a sitemap.
+
+## While you are getting started
+
+To all newcomers: please read the section [New User Tutorial]({{base}}/tutorial). 
+That section provides you with a step-by-step instructions for your first easy setup. 
+
+OpenHAB runs on most popular platforms such as Linux, Windows and MacOS and on almost any hardware ranging from Raspberry Pis to desktop computers and server PCs. 
+You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/index.html) article. 
+If you have a strong preference towards a particular platform, then that platform is probably your best choice.
+
+You can install openHAB on your desktop computer for evaluation purposes if you already have any of these systems available for use, but we recommend using a dedicated system in the long run. 
+If you feel serious about home automation it may be better to start with a dedicated system right away.
+
+For anyone undecided: many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi (Version 2  or 3)](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hassle-free openHAB setup". 
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system easily capable of automating your entire home.
```

Comment:
I'm not sure what you want to say with "While... it is a complete openHAB home automation system". Instead maybe mention that it is based on a normal Linux distribution thus helping users from the beginning to the full-blown setup

Reply:
```suggestion
openHABian offers a streamlined and simplified way to get up and running quickly and provides a complete openHAB home automation system easily capable of automating your entire home.
```

Reply:
Fixing sentence, hope that is acceptable for both of you